ui/shared/LogoMenu.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const choices = [
+  { url: '/', text: 'Treeherder' },
+  { url: '/perf.html', text: 'Perfherder' },
+  { url: '/intermittent-failures.html', text: 'Intermittent Failures' },
+];
+
+export default function LogoMenu(props) {
+  const { menuText, menuTitle, menuImage } = props;
+
+  const menuChoices = choices.filter(choice => choice.text !== menuText);
+
+  return (
+    <span className="dropdown">
+      <button
```

Comment:
I tried uncommenting the disabled ESLint rules and scanning just this file - and got:

```
error: Missing an explicit type attribute for button (react/button-has-type) at ui\shared\LogoMenu.jsx:17:7:
  15 |   return (
  16 |     <span className="dropdown">
> 17 |       <button
     |       ^
  18 |         id="th-logo"
  19 |         title={menuTitle}
  20 |         data-toggle="dropdown"
```

Whilst this rule isn't yet enabled and this was mostly just a file move, I think it would be good to try and ensure any newly added files don't regress the disabled rules any further - if you'd be ok with fixing this one?

Reply:
You bet, no problem.